delfin/task_manager/scheduler/schedule_manager.py

```diff
@@ -76,6 +79,15 @@ def schedule_boot_jobs(self):
                 LOG.error("Failed to initialize periodic tasks, reason: %s.",
                           six.text_type(e))
                 raise e
+        metrics_task_server = service. \
```

Comment:
This sub routine is started only by leader.. if we keep metrics_task_service here only leader will have this service running..

Reply:
Created a new service here, to get notification when new job created.